recipes/sklearn-contrib-lightning/build.sh

```diff
@@ -0,0 +1 @@
+$PYTHON setup.py install     # Python command to install the script.
```

Comment:
@fabianp you don't need the `bld.bat` and `build.sh` if the build script is a single and simple line. (See below.)